docs/add-ins/sideload-outlook-add-ins-for-testing.md

```diff
@@ -15,17 +15,52 @@ You can use sideloading to install an Outlook add-in for testing without having
 
 ## Sideload an add-in in Outlook in Office 365
 
+The process for sideloading an add-in in Outlook in Office 365 depends upon whether you are using the new Outlook on the web or classic Outlook on the web.
+
+- If your mailbox toolbar looks like the following image, see [Sideload an add-in in the new Outlook on the web](#sideload-an-add-in-in-the-new-outlook-on-the-web).
+
+    ![partial screenshot of the new Outlook on the web toolbar](images/outlook-on-the-web-new-toolbar.png)
+
+- If your mailbox toolbar looks like the following image, see [Sideload an add-in in classic Outlook on the web](#sideload-an-add-in-in-classic-outlook-on-the-web).
+
+    ![partial screenshot of the classic Outlook on the web toolbar](images/outlook-on-the-web-classic-toolbar.png)
+
+> [!NOTE]
+> If your organization has included its logo in the mailbox toolbar, you might see something slightly different than shown in the preceding images.
+
+### Sideload an add-in in the new Outlook on the web
+
 1. Go to [Outlook in Office 365](https://outlook.office.com).
 
-1. Choose the gear icon in the top-right section and select **Manage add-ins**.
+1. In Outlook on the web, create a new message or reply to an existing message.   
+
+1. Choose **...** from the bottom of the new message window and then select **Get Add-ins** from the menu that appears.
```

Comment:
I suggest we remove "window" to account for a reply to an existing message which doesn't necessarily go to a new window.

Reply:
Done.